cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -413,18 +413,75 @@ func metricsListGrabAggregateConvert(ctx context.Context, t metricsTarget) (map[
 	if len(targetList) > 1 {
 		resetDetected = false
 	}
-
+	
 	return families, interestingMetrics, resetDetected, nil
 }
 
 // Handler grabs metrics from target, aggregates the metrics and convert them into cloudwatch metrics
 func Handler(ctx context.Context, t metricsTarget) {
 	families, interestingMetrics, resetDetected, err := metricsListGrabAggregateConvert(ctx, t)
 
+	updatePromMetrics(interestingMetrics)
```

Comment:
I think we should update prometheus metrics after CloudWatch metrics are updated, and we should do so only when env var is true. We should also add a comment explaining the call.